### ISSUE TITLE
Failure to install: "cannot remove prior installation of package"

### ISSUE THREAD

**Issue by gaiteiro** [rmcelreath/rethinking#233]
Windows 10, R 4.0.0, RTool40, RStan just installed....using RGui, having installed devtools (and having had to install package 'ps' separately):

> rstan_options(auto_write = TRUE)
> options(mc.cores = parallel::detectCores())
> devtools::install_github("rmcelreath/rethinking")
Downloading GitHub repo rmcelreath/rethinking@master
These packages have more recent versions available.
It is recommended to update all of them.
Which would you like to update?

1: All                                              
[which I selected. Then:]

Enter one or more numbers, or an empty line to skip updates:
1
glue      (1.4.0        -> 1.4.1       ) [CRAN]
scales    (1.1.0        -> 1.1.1       ) [CRAN]
ellipsis  (0.3.0        -> 0.3.1       ) [CRAN]
vctrs     (0.2.4        -> 0.3.0       ) [CRAN]
pkgbuild  (c61bb8068... -> 609321c36...) [GitHub]
backports (1.1.6        -> 1.1.7       ) [CRAN]
Installing 5 packages: glue, scales, ellipsis, vctrs, backports

  There are binary versions available but the source versions are later:
          binary source needs_compilation
glue       1.4.0  1.4.1              TRUE
ellipsis   0.3.0  0.3.1              TRUE
backports  1.1.6  1.1.7              TRUE

trying URL 'https://www.stats.bris.ac.uk/R/bin/windows/contrib/4.0/scales_1.1.1.zip'
Content type 'application/zip' length 561199 bytes (548 KB)
downloaded 548 KB

trying URL 'https://www.stats.bris.ac.uk/R/bin/windows/contrib/4.0/vctrs_0.3.0.zip'
Content type 'application/zip' length 1088566 bytes (1.0 MB)
downloaded 1.0 MB

package ‘scales’ successfully unpacked and MD5 sums checked
package ‘vctrs’ successfully unpacked and MD5 sums checked
Error: Failed to install 'rethinking' from GitHub:
  (converted from warning) cannot remove prior installation of package ‘vctrs’

I tried selecting various options, replying yes/no when asked if I wanted to download from source. When selecting '1' then 'No' and also when selecting '3' (no updates) I got a slightly different error:

package ‘glue’ successfully unpacked and MD5 sums checked
Error: Failed to install 'rethinking' from GitHub:
  (converted from warning) cannot remove prior installation of package ‘glue’

.....can anyone help??

**Comment by jamesbrownlow** [comment]
I had the same experience... I haven't done anything yet, except I'm using
STAN in Python

**Comment by gaiteiro** [comment]
I eventually solved the problem by manually deleting the folders from R\R-4.0.0\library
....I then had to install the packages e.g.
install.packages("glue")
...before beginning again with the rethinking package. Along the way, I had to manually delete several 00LOCK- folders left behind in R\R-4.0.0\library by failed install attempts, as apparently these prevent reinstallation.

I should have stuck to PyStan like @jamesbrownlow! I thought it was a struggle getting stuff working with Python, but R takes it to a different level. Perhaps this is something like a mediaeval guild system, in which coders keep out potential competitors until they have served their apprenticeship.
